packages/migrate/src/Migrate.ts

```diff
@@ -26,26 +26,30 @@ interface MigrateSetupInput {
   migrationsDirPath?: string
   enabledPreviewFeatures?: string[]
   schemaContext?: SchemaContext
+  schemaFilter?: MigrateTypes.SchemaFilter
```

Comment:
Nit: Does `schemaFilter` *really* need to be `nullable` or undefined? This will cause a chain of `schemaFilter?['prop'] property key access when trying to read such filter.

It seems more explicit to me to pass a default filter object, like the following:
```typescript
const defaultSchemaFilter = {
  externalTables: [],
  // in the future:
  exclude: [],
} satisfies MigrateTypes.SchemaFilter
```

Reply:
In general, if we can avoid introducing `null`s, let's not.

Reply:
I might change the interface between TS and Rust again in a dedicated PR to remove the optionality in the input types there. Was using optional there at first because it might ease the adoption (similar to how you would add a new field to a REST api).

Reply:
See the changes I just pushed